update.sh

```diff
@@ -22,7 +22,7 @@ declare -A base=(
 latest="$(
 	git ls-remote --tags https://github.com/roundcube/roundcubemail.git \
 		| cut -d/ -f3 \
-		| grep -P -- '^[\d\.]+$' \
+		| grep -E -- '^([0-9]{1,}\.)+[0-9]{1,}$' \
 		| sort -V \
 		| tail -1
 )"
```

Comment:
The latest stable version can be fetched from here:
```
latest="$(curl -s http://roundcube.net/VERSION.txt)"
```

Reply:
Thanks, changed in https://github.com/roundcube/roundcubemail-docker/pull/20/commits/4bace5ffed724f23600e11a5aefcaf0a82eb98fd